OrbitGl/FunctionsDataView.cpp

```diff
@@ -145,39 +138,30 @@ void FunctionsDataView::OnContextMenu(const std::string& action, int menu_index,
                                       const std::vector<int>& item_indices) {
   if (action == kMenuActionSelect) {
     for (int i : item_indices) {
-      GOrbitApp->SelectFunction(GetFunction(i));
+      GOrbitApp->SelectFunction(*GetFunction(i));
     }
   } else if (action == kMenuActionUnselect) {
     for (int i : item_indices) {
-      GOrbitApp->DeselectFunction(GetFunction(i));
+      GOrbitApp->DeselectFunction(*GetFunction(i));
     }
   } else if (action == kMenuActionDisassembly) {
-    int32_t pid = GOrbitApp->GetSelectedProcessId();
     for (int i : item_indices) {
-      GOrbitApp->Disassemble(pid, GetFunction(i));
+      GOrbitApp->Disassemble(GOrbitApp->GetSelectedProcess()->pid(), *GetFunction(i));
     }
   } else {
     DataView::OnContextMenu(action, menu_index, item_indices);
   }
 }
 
 void FunctionsDataView::DoFilter() {
-  // TODO(antonrohr) This filter function can take a lot of time when a large
```

Comment:
Is that comment not relevant anymore?

Reply:
When I wrote this comment a while ago, I used a debug build instead of release and then it applied. I did not realize at this time that it is only horribly slow when on debug.